docs/source/markdown/podman-build.1.md

```diff
@@ -940,22 +940,26 @@ $ podman build -f dev/Containerfile https://10.10.10.1/podman/context.tar.gz
 
 ## Files
 
-### `.dockerignore`
+### .containerignore/.dockerignore
 
-If the file .dockerignore exists in the context directory, `buildah copy` reads
-its contents. Use the `--ignorefile` flag to override .dockerignore path location.
+If the file *.containerignore* or *.dockerignore* exists in the context directory,
+`podman build` reads its contents. Use the `--ignorefile` flag to override the
+.containerignore path location.
 Podman uses the content to exclude files and directories from the context
 directory, when executing COPY and ADD directives in the
 Containerfile/Dockerfile
 
-Users can specify a series of Unix shell globals in a .dockerignore file to
+The .containerignore and .dockerignore files have the same syntax; if both
+are in the context directory, podman build will use .containerignore.
```

Comment:
Suggestion: '...will use __only__ .containerignore` (boldface mine for emphasis; not needed in actual document). Without 'only', it is ambiguous whether the sentence means "will use .containerignore in addition to .dockerignore".